gulpfile.js

```diff
@@ -148,7 +148,9 @@ gulp.task('installFixtures', function() {
     }, 1 * 1000);
     shell.cd('test/fixtures');
 
-    execAsync('npm install --quiet', {cwd: '../fixtures'}).then(() => {
+    execAsync('type yarn &> /dev/null | yarn install || npm install --quiet', {
```

Comment:
we should be checking if the user is on Windows, and using the proper Windows commands if so